code/mob/living/critter/flockcritter_parent.dm

```diff
@@ -105,6 +105,7 @@
 		src.visible_message("<span class='notice'>[src] harmlessly absorbs [P].</span>")
 		return FALSE
 	..()
+	return TRUE
```

Comment:
```suggestion
	return ..()
```

Reply:
I think it makes sense for flockdrones to still be offended by being attacked even if the shot reflects off them? (don't actually think this can happen currently, but someone said something about making flockdrones reflect phaser shots)

Reply:
They can't right now, but reflected shots when testing were still hitting if I remember right